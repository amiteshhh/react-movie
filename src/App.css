.App {
  background-color: rgb(48, 48, 48);
  color: rgb(255, 255, 255);
  min-height: 100vh;
  display: flex;
  flex-direction: column;
  padding: 10px 50px
}

.App-message-container {
  display: flex;
  color: rgb(255, 255, 255);
  background: rgb(66, 66, 66);
  align-items: center;
  margin: 10px;
  padding: 16px;
  box-shadow: rgba(0, 0, 0, 0.2) 0px 2px 1px -1px, rgba(0, 0, 0, 0.14) 0px 1px 1px 0px, rgba(0, 0, 0, 0.12) 0px 1px 3px 0px;
  border-radius: 4px;
}

.App-message-heading {
  margin: 0;
}

.App-message {
  margin: 0 10px;
}