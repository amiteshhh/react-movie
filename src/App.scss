@import '~bootstrap/scss/bootstrap';
@import './styles/index.scss';

.app {
  background-color: $bodyBg;
  min-height: 100vh;
  display: flex;
  flex-direction: column;
  padding: 10px 50px;
}

.app-header {
  position: absolute;
  left: 90px;
  top: 20px;
}

.app-footer {
  text-align: center;
  padding: 15px;
  background: rgba(52, 58, 64, 0.55);
  font-size: 120%;
}

.margin-0-auto {
  margin: 0 auto;
}