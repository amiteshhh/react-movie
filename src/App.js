import React from 'react';
import './App.css';

const envName = process.env.production ? 'production' : 'dev';

function App() {
  return (
    <div className="app">
      <h3>
        App is running in
        <code className="env-label">{envName}</code>
        environment.
      </h3>
      Hello react webpack!
    </div>
  );
}

export default App;
