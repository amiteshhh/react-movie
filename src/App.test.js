import React from 'react';
import { render } from '@testing-library/react';
import renderer from 'react-test-renderer';
import App from './App';

describe('App component', () => {

  test('renders hello message', () => {
    const { getByText } = render(<App />);
    const helloElement = getByText(/hello react/i);
    expect(helloElement).toBeInTheDocument();
  });

  it('renders correctly', () => {
    const component = renderer
      .create(<App />)
      .toJSON();
    expect(component).toMatchSnapshot();
  });

});
