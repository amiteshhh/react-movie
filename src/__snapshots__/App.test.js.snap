// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component renders correctly 1`] = `
<div
  className="app"
>
  <h3>
    App is running in
    <code
      className="env-label"
    >
      dev
    </code>
    environment.
  </h3>
  Hello react webpack!
</div>
`;
